android/src/main/java/com/oijusti/firebaseuiauth/RNFirebaseuiAuthModule.java

```diff
@@ -134,6 +135,10 @@ else if (customization.equals("logo")) {
       } catch (PackageManager.NameNotFoundException e) { }
     }
 
+    if (allowAutoUpgradeAnonymousUsers) {
+      builder = builder.enableAnonymousUsersAutoUpgrade();
```

Comment:
We can remove the variable assignment and leave it just like:
`      builder.enableAnonymousUsersAutoUpgrade();`